addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2370,6 +2370,10 @@ export class OdooEditor extends EventTarget {
             hint.classList.remove('oe-hint', 'oe-command-temporary-hint');
             hint.removeAttribute('placeholder');
         }
+        const externalLinkSelector = `a:not([href^="${location.origin}"]):not([href^="/"])`;
+        for (const externalLink of this.editable.querySelectorAll(externalLinkSelector)) {
+            externalLink.setAttribute('target', '_blank');
+        }
```

Comment:
I'm surprised to see the change here. Shouldn't that be done in the code that generates the link instead ? I remember there used to be a "open in new tab" checkbox in the link options for example. If it is still there then we could simply check it by default ?

Reply:
Or the alternative that was actually proposed in the task: change the widget HTML so that clicking a link in formview readonly opens in target new

Reply:
> Or the alternative that was actually proposed in the task: change the widget HTML so that clicking a link in formview readonly opens in target new

The problem is I don't think you can do that in Javascript without it being technically a popup (and so being potentially blocked). Or you have another idea?